front_office/products_page/templates/products_page.html

```diff
@@ -0,0 +1,72 @@
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+        <meta name="description" content="" />
+        <meta name="author" content="" />
+        <!-- Favicon-->
+        <link rel="icon" type="image/x-icon" href="assets/favicon.ico" />
+        <!-- Bootstrap icons-->
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css" rel="stylesheet" />
+        <!-- Core theme CSS (includes Bootstrap)-->
+        <link href="css/styles.css" rel="stylesheet" />
+    </head>
+    <body>
+        <!-- Section-->
+        <section class="py-5">
+            <div class="container px-4 px-lg-5 mt-5">
+                <div class="row gx-4 gx-lg-5 row-cols-2 row-cols-md-3 row-cols-xl-4 justify-content-center">
+                    {% for product in products %}
+                    <div class="col mb-5">
+                        <div class="card h-100">
+                            <!-- Product image-->
+                            <a href="{% url 'front_office.pdp_page' product.id %}" style="color: rgb(53, 52, 52); text-decoration: none;">
+                            <img class="card-img-top" src="{{ product.image.url }}" alt="..." />
+                            <!-- Product details-->
+                            <div class="card-body p-4">
+                                <div class="text-center">
+                                    <!-- Product name-->
+                                    <h5 class="fw-bolder"> {{ product.name }} </h5></a>
+                                    <!-- Product brand-->
+                                    Manufacturer: {{ product.brand }}<br>
+                                    <h7>Availibility: 
+                                        {% if product.in_stock %}
+                                        <span class="text-success">                                        
+                                        In stock</span>
+                                        {% else %}
+                                        <span class="text-danger">Out of stock</span>
+                                        {% endif %}
+                                    </h7><br><br>
+                                    <!-- Product price-->
+                                    <h5>Price: €{{ product.price }}</h5>
+                                </div>
+                            </div>
+                            <!-- Product actions-->
+                            <div class="card-footer p-4 pt-0 border-top-0 bg-transparent">
+                                <div class="text-center"><a class="btn btn-success mt-auto" href="#">Add to cart</a></div>
+                            </div>
+                        </div>
+                    </div>
+                    {% endfor %}
+                </div>
+            </div>
+        </section>
+        <!-- Footer-->
+        <footer class="py-5 bg-light">
```

Comment:
Footer will be done by separate ticket to be re-usable instead of being on this one page